client/src/components/helpers/loader.test.tsx

```diff
@@ -1,11 +1,9 @@
-import { render, cleanup, screen } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import React from 'react';
 
 import Loader from './loader';
 
 describe('<Loader />', () => {
-  afterEach(cleanup);
```

Comment:
`cleanup` is performed automatically by your test runner, you don't need manual cleanups